phpunit.xml

```diff
@@ -8,6 +8,7 @@
     <testsuites>
         <testsuite name="main">
             <directory>tests</directory>
+            <directory>rules-tests</directory>
```

Comment:
@TomasVotruba here `rules-tests` need to be registered to show the PHPUnit error, it was never executed.

Reply:
Damn, that's a big miss from my side 🤦 😅